projects/fhi-angular-highcharts/src/lib/fhi-angular-highcharts.component.ts

```diff
@@ -344,6 +367,16 @@ export class FhiAngularHighchartsComponent implements OnChanges {
       });
   }
 
+  private canShowDownloadButton(): boolean {
```

Comment:
Om du returnerer `!!this.diagramOptions.controls?.downloadButton?.show &&
      this.diagramOptions.activeDiagramType !== 'table'` direkte så tror jeg du oppnår det samme